README.md

```diff
@@ -113,7 +115,22 @@ Documentation (currently mostly API documentation unfortunately) is available on
 
 https://ipfs.io/ipns/QmZ86ow1byeyhNRJEatWxGPJKcnQKG7s51MtbHdxxUddTH/Software/Python/ipfsapi/
 
-The [`ipfs` command-line Client documentation](https://ipfs.io/docs/commands/) may also be useful in some cases.
+The `ipfs` [command-line Client documentation](https://ipfs.io/docs/commands/) may also be useful in some cases.
+
+### Important changes from `ipfsApi 0.2.x`
+
+ * The Python package has been renamed from `ipfsApi` to `ipfsapi`
+ * The PIP module has been renamed from `ipfs-api` to `ipfsapi` (please update your requirement files)
+ * A lot of changes in the internal code
+    - Commands have been completely removed
+    - Usage of `requests` or other libraries is considered an implementation detail from now on
```

Comment:
Was it not considered an implementation detail before? I'm not sure that I understand what you mean.